src/codeu/chat/client/simplegui/ChatSimpleGui.java

```diff
@@ -25,36 +25,34 @@
 import codeu.chat.client.View;
 import codeu.chat.util.Logger;
 
-// Chat - top-level client application - Java Simple GUI (using Java Swing)
+/**
+ * Chat - top-level client application - Java Simple GUI (using Java Swing)
+ */
 public final class ChatSimpleGui {
-
-    UserMap userMap = new UserMap();    // creates new worthless map
-
-    private final static Logger.Log LOG = Logger.newLog(ChatSimpleGui.class);
-
-    public static JFrame mainFrame;
-
+    ///////////////////////
+    // PRIVATE VARIABLES //
+    ///////////////////////
+    private static JFrame mainFrame;
```

Comment:
Yes, static means class-level variable, which means ALL instances of the class will share the same variable (i.e. there exists ONLY ONE copy).  You can view it as being able to get it without creating an instance, but should also be very careful:

public final class SomeClass {
private static int foo = 3;
public void changeFoo(int newFoo) {
foo = newFoo;
}
public int getFoo() {
return foo;
}
}
SomeClass c1 = new SomeClass();
SomeClass c2 = new SomeClass();
c1.getFoo(); // returns 3
c2.getFoo(); // also returns 3
c1.setFoo(15);
c1.getFoo(); // returns 15
c2.getFoo(); // will also return 15 now